src/pdf/DriverOverview/DriverOverviewPdfButton.tsx

```diff
@@ -102,6 +114,7 @@ const getDriverPdfData = async (parcelIds: string[]): Promise<DriverPdfResponse>
             contact: client?.phone_number ?? "",
             packingDate: formatDateToDate(parcel.packing_date) ?? null,
             instructions: client?.delivery_instructions ?? "",
+            numberOfLabels: parcel.labelCount ?? null,
```

Comment:
If you make all the types null as per the other comment you can probably leave this as just `parcel.labelCount`. Although being explicit doesn't hurt - up to you what you want to do with it